TASK/TaskVehicleTempAction.md

```diff
@@ -21,22 +21,24 @@ void TASK_VEHICLE_TEMP_ACTION(Ped driver, Vehicle vehicle, int action, int time)
 '11 - turn right + restore wheel pos to center in the end
 '13 - turn left + go reverse
 '14 - turn left + go reverse
-'16 - crash the game after like 2 seconds :)
-'17 - keep actual state, game crashed after few tries
-'18 - game crash
-'19 - strong brake + turn left/right
-'20 - weak brake + turn left then turn right
-'21 - weak brake + turn right then turn left
-'22 - brake + reverse
-'23 - accelerate fast
-'24 - brake
-'25 - brake turning left then when almost stopping it turns left more
-'26 - brake turning right then when almost stopping it turns right more
-'27 - brake until car stop or until time ends
-'28 - brake + strong reverse acceleration
+'16 - plane fly up (crashes game if not in plane) :)
+'17 - plane fly straight (crashes game if not in plane)
+'18 - plane sharp left (crashes game if not in plane)
+'19 - plane sharp right (crashes game if not in plane)
+'20 - strong brake + turn left/right
+'21 - weak brake + turn left then turn right
+'22 - weak brake + turn right then turn left
+'23 - brake + reverse
+'24 - accelerate fast
+'25 - brake
+'26 - brake turning left then when almost stopping it turns left more
+'27 - brake turning right then when almost stopping it turns right more
+'28 - brake until car stop or until time ends
+'29 - brake + strong reverse acceleration
 '30 - performs a burnout (brake until stop + brake and accelerate)
 '31 - accelerate + handbrake
 '32 - accelerate very strong
+'33 - Surface in submarine
 Seems to be this:
 Works on NPCs, but overrides their current task. If inside a task sequence (and not being the last task), "time" will work, otherwise the task will be performed forever until tasked with something else
 ```
```

Comment:
This should be re-written, documentation for end-users should be factual 😊

Reply:
Hey! That one wasn't me! But nonetheless, I'll try to update it so it's a bit more logical!